pkg/services/rpcsrv/server.go

```diff
@@ -155,12 +157,14 @@ type (
 		notificationSubs  int
 		transactionSubs   int
 		notaryRequestSubs int
+		headerSubs        int
 
 		blockCh           chan *block.Block
 		executionCh       chan *state.AppExecResult
 		notificationCh    chan *state.ContainedNotificationEvent
 		transactionCh     chan *transaction.Transaction
 		notaryRequestCh   chan mempoolevent.Event
+		headerCh          chan *block.Header
```

Comment:
Rename to `blockHeaderCh` and `blockHeaderSubs` and move under `blockCh` and `blockSubs` resp.